EosPayload/drivers/radio_driver.py

```diff
@@ -121,7 +124,7 @@ def xbee_send_callback(_client, _userdata, message):
                 # Store to data file
                 if self.log_lock.acquire(blocking=False):
                     try:
-                        self.data_log(["sent", packet_from_mqtt.encode_to_string()])
+                        self.data_log(["sent"])
```

Comment:
Change this to "Sent packet". Also add this above it `# TODO change this to include packet as a string (encode_to_string function) `